test/basic.js

```diff
@@ -24,6 +24,10 @@ test('basic normalization tests', function (t) {
     { url: 'git@github.com:npm/npm.git', branch: 'v1.0.27' }
   )
   t.same(
+    normalize('git+ssh://git@github.com:/npm/npm.git#v1.0.28'),
```

Comment:
Absolutely! scp-style syntax requires the semicolon. That's the only way for it to know that an absolute URL is remote:

`scp git@myhost/x/y => ~error~`

That said, `ssh://` *is* able to make ssh connections without the scp-style syntax.